Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -15,21 +29,41 @@ final class PledgeViewController: UIViewController {
     PledgeContinueViewController.instantiate()
   }()
 
+  private lazy var pledgeDescriptionSectionViews = {
+    [self.pledgeDescriptionViewController.view, self.descriptionSectionSeparator]
+  }()
+
   private lazy var pledgeDescriptionViewController = {
     PledgeDescriptionViewController.instantiate()
   }()
 
-  private lazy var pledgeSummaryViewController = {
-    PledgeSummaryViewController.instantiate()
+  private lazy var pledgeInputsSectionViews = {
+    [self.pledgeAmountViewController.view, self.pledgeShippingLocationViewController.view]
+  }()
+
+  private lazy var pledgeLoginSectionViews = {
+    [self.pledgeContinueViewController.view]
+  }()
+
+  private lazy var pledgePaymentMethodsSectionViews = {
+    [self.pledgePaymentMethodsViewController.view]
```

Comment:
Are we doing arrays like this in anticipation of a separator potentially being added later on?

Reply:
Yes exactly, but also for consistency so that we carry the concept of "sections" as being arrays of views for every logical grouping in the screen.

Reply:
Just had a quick peek at the naming conventions here...would it make sense to simplify the naming by dropping `pledge` prefix? We already know we're working with `PledgeVC` so it could simplify a lot of naming here?

Reply:
Open to doing that. Thoughts @justinswart ?

Reply:
Hmm, I know it's a bit verbose but I typically like the variable name to reflect the type name.

I'm more interested in these potentially not being lazy vars and constructed where we use them?

Reply:
oh I see what @dusi meant, yeah for things like `pledgePaymentMethodsSectionViews` that could just be `paymentMethodsSectionViews`

Reply:
ah i'm not using github properly, i saw your comment below now about why we store a reference 👍